.github/CODEOWNERS

```diff
@@ -2,7 +2,7 @@
 * @carryall
 
 # Team Members
-* @bterone @hanam1ni @hoangmirs @malparty @rosle @tyrro
+* @carryall @bterone @hanam1ni @hoangmirs @malparty @rosle @tyrro
```

Comment:
We had a small discussion here: https://github.com/nimblehq/gin-templates/pull/75/files#r841554757

And this is the suggestion from Xavier and Olivier:
```suggestion
# Team
# @carryall is the Team Lead and the others are team members
* @carryall @bterone @hanam1ni @hoangmirs @malparty @rosle @tyrro
```

With that, we should remove the first lines as well

Reply:
Thanks @hoangmirs updated in [f5f2f6e](https://github.com/nimblehq/react-templates/pull/68/commits/f5f2f6e9f1d0cd91434262ccf5b346a5ead0601f)